_component_design/help-beacon.md

```diff
@@ -1,15 +1,14 @@
 ---
-title: Help Beacon
-parent: Help
-secondary: Notifications
+title: Beacon
+parent: Notifications
 layout: component
 category: Components
 usage: >
-  The beacon can be used to identify the location of products or features in the control panel. It can also be used to indicate advanced functionality to existing features
+  A beacon is an animation that attracts a user's attention to a new feature or advanced functionality in a control panel. A help beacon disappears after a user clicks it.
```

Comment:
Seems like you're not calling it a "help beacon" anymore, so the second sentence should be "A beacon disappears after a user clicks it."